### PR TITLE
feat: Import improvements

### PR DESCRIPTION
**Pull request by JanCizmar**

### CODE REVIEW

File: backend/app/src/test/kotlin/io/tolgee/api/v2/controllers/v2ImportController/V2ImportControllerAddFilesTest.kt [hunk 1]
```diff
@@ -135,6 +142,34 @@ class V2ImportControllerAddFilesTest : AuthorizedControllerTest() {
       }
   }
 
+  @Test
+  fun `it imports empty keys`() {
```
Comment (cyyynthia) [inline backend/app/src/test/kotlin/io/tolgee/api/v2/controllers/v2ImportController/V2ImportControllerAddFilesTest.kt:146]: Are we sure we want to keep this considering the UI issues it causes?

Comment (JanCizmar) [reply backend/app/src/test/kotlin/io/tolgee/api/v2/controllers/v2ImportController/V2ImportControllerAddFilesTest.kt:146]: Importing only empty keys is edge case. So I would keep it. In most cases user imports also some translations.
